src/main/java/io/reactivex/Scheduler.java

```diff
@@ -368,7 +368,7 @@ public long now(@NonNull TimeUnit unit) {
 
             @Override
             public void run() {
-                decoratedRun.run();
+                getWrappedRunnable().run();
```

Comment:
Please don't turn these into method calls.

Reply:
Could you take me small tip? Without invoking getWrappedRunnable I don't have a 100% code coverage in Scheduler.java because I'm using the method only in RxJavaPlugins which has 100% code coverage.

Reply:
When this type of task is created and returned, then invoke `unwrapRunnable` on it.

Reply:
Do you mean something like that?
```
@NonNull
    public Disposable scheduleDirect(@NonNull Runnable run, long delay, @NonNull TimeUnit unit) {
        final Worker w = createWorker();
        final Runnable decoratedRun = RxJavaPlugins.onSchedule(run);
        DisposeTask task = new DisposeTask(decoratedRun, w);
        w.schedule(task, delay, unit);
        return RxJavaPlugins.unwrapRunnable(task);
    }
```
but in that case I have incompatible types Disposable and Runnable

Reply:
No, do the unwrapping in the unit test. Also for convenience, the unwrapRunnable should perhaps accept `Object` instead of `Runnable`.